### ISSUE TITLE
I can't edit with VIA, rev1.1?

### ISSUE THREAD

**Issue by ElRisho** [ajmandourah/Ladispad#6]
I usually use .json in via to edit my macropad, via got updated and now I can't edit my macropad. I'm not a programmer and I've tried to edit the code in the repository but I can't get it.
Pls help:(

https://i.gyazo.com/2c2088e6e53f5dda72d3041ee5fd3f14.png
https://i.gyazo.com/9896bb3a13d67d3d1bf5cb49162f54a7.png
https://i.gyazo.com/144cd175a9ff77a003f3a4aadd46edc7.png

**Comment by ajmandourah** [comment]
Seems like VIA is rejecting the vendorID for some reason . 
try removing the line 
`#define VENDOR_ID       0xFEED`
in the config file and recompile .. report if it helps and in the via app you can load the json via the " load draft definition "

**Comment by ElRisho** [comment]
The same thing happens, It gives me some error that I don't understand without 
`#define VENDOR_ID       0xFEED` or `#define VENDOR_ID       0xF3ED`  

(0xF3ED for putting something)

I think the problem is in vendor ID is 0xFEED.
Because in the .json when changing "0xFEED" to something different it recognizes it, but it doesn't recognize the macropad
https://i.gyazo.com/ddfc82254ab8c0c9089e19cb68f54f97.png
https://i.gyazo.com/7ac1d3700ea8a6303d0ff118f30b0443.png

**Comment by ajmandourah** [comment]
Yes Via confirm that they do not allow FEED as a vendor ID. Changing it should work.

What do you mean by not recognizing? Is it even after changing the vendor ID?
Try changing the keys inside via and test out.

Sorry I lost my ladispad but I probably have it somewhere around. I will test out if I find it.

**Comment by ElRisho** [comment]
It doesn't recognize it with the default '.hex'
Changing the vendor ID doesn't compile and I don't know what I'm doing wrong, even if I change the 0xFEED to something else, look at this https://i.gyazo.com/31dfdd6b64391f5290079e30fe25ce76.png
If you find it and you can help me i appreciate it xD

**Comment by ElRisho** [comment]
It happened to another person and another solution would be VIA version 1.3.1 but I can't find it anywhere, not even in the repository
https://github.com/the-via/releases/issues/247#issuecomment-1430728778
